### ISSUE TITLE
FSL compatibility documentation

### ISSUE THREAD

**Issue by Lestropie** [MRtrix3/mrtrix3#515]
I'm getting questions on this more and more, and finding out about more people that have been handling gradient table data incorrectly. So I think we need a full discussion topic page in the documentation on this.

- bvecs / bvals vs. MRtrix gradient table
  - NOT just a reformatting
  - scanner vs. image space
- NiFTI data strides
  - Including order of performing this relative to bvecs / bvals export
- qform / sform matrices: What MRtrix reads & writes

**Comment by thijsdhollander** [comment]
Yep, fully agreed.  We could also clarify the bvecs/bvals thing slightly better in the help/doc for those options.  At the moment, coming in without any prior knowledge, you might indeed naively think it's just a different formatting.

**Comment by jdtournier** [comment]
:+1:

It's going to need lots of diagrams to really explain all this clearly...

**Comment by Lestropie** [comment]
The `bvecs` / `bvals` side of things should be dealt with in #932. Does more need to be done in terms of `qform` / `sform` / generating diagrams to explain the difference between image and scanner space?
